intermediate_source/quantized_transfer_learning_tutorial.rst

```diff
@@ -0,0 +1,510 @@
+        nn.Flatten(1),
+        new_head,
+      )
+      return new_model
+
+.. warning:: Currently the quantized models can only be run on CPU.
+  However, it is possible to send the non-quantized parts of the model to a GPU.
+
+.. code:: python
+
+    import torch.optim as optim
+    new_model = create_combined_model(model_fe)
+    new_model = new_model.to('cpu')
+
+    criterion = nn.CrossEntropyLoss()
+
+    # Note that we are only training the head.
+    optimizer_ft = optim.SGD(new_model.parameters(), lr=0.01, momentum=0.9)
+
+    # Decay LR by a factor of 0.1 every 7 epochs
+    exp_lr_scheduler = optim.lr_scheduler.StepLR(optimizer_ft, step_size=7, gamma=0.1)
+
+
+Train and evaluate
+~~~~~~~~~~~~~~~~~~
+
+This step takes around 15-25 min on CPU. Because the quantized model can
+only run on the CPU, you cannot run the training on GPU.
+
+.. code:: python
+
+    new_model = train_model(new_model, criterion, optimizer_ft, exp_lr_scheduler,
+                            num_epochs=25, device='cpu')
+
+    visualize_model(new_model)
+    plt.tight_layout()
+
+
+Part 2. Finetuning the Quantizable Model
+----------------------------------------
+
+In this part, we fine tune the feature extractor used for transfer
+learning, and quantize the feature extractor. Note that in both part 1
+and 2, the feature extractor is quantized. The difference is that in
+part 1, we use a pretrained quantized model. In this part, we create a
+quantized feature extractor after fine tuning on the data-set of
+interest, so this is a way to get better accuracy with transfer learning
+while having the benefits of quantization. Note that in our specific
+example, the training set is really small (120 images) so the benefits
+of fine tuning the entire model is not apparent. However, the procedure
+shown here will improve accuracy for transfer learning with larger
+datasets.
+
+The pretrained feature extractor must be quantizable.
+To make sure it is quantizable, perform the following steps:
+
+ 1. Fuse ``(Conv, BN, ReLU)``, ``(Conv, BN)``, and ``(Conv, ReLU)`` using
+    ``torch.quantization.fuse_modules``.
+ 2. Connect the feature extractor with a custom head.
+    This requires dequantizing the output of the feature extractor.
+ 3. Insert fake-quantization modules at appropriate locations
+    in the feature extractor to mimic quantization during training.
+
+For step (1), we use models from ``torchvision/models/quantization``, which
+have a member method ``fuse_model``. This function fuses all the ``conv``,
+``bn``, and ``relu`` modules. For custom models, this would require calling
+the ``torch.quantization.fuse_modules`` API with the list of modules to fuse
+manually.
+
+Step (2) is performed by the ``create_combined_model`` function
+used in the previous section.
+
+Step (3) is achieved by using ``torch.quantization.prepare_qat``, which
+inserts fake-quantization modules.
+
+
+As step (4), you can start "finetuning" the model, and after that convert
```

Comment:
Minor comment, suggest replacing with:
Once steps  1-3 are done, we can start fine tuning the model (step 4), followed by converting the model into a quantized. (step 5).